amnestywebsite/humanity-theme
# Adds the full width image with buttons block pattern

Ref: https://github.com/amnestywebsite/humanity-theme/issues/178

Adds the full width image with buttons block pattern

**Steps to test**:
1. Edit a post
2. Open the block inserter and check the new pattern appears under the patterns tab
3. Insert the pattern and check it matches the example in the ticket

Example: http://bigbite.im/i/Y1dDpy
